cloud_aggregator/ingest/ingest.py

```diff
@@ -26,18 +21,13 @@ def handler(event, context):
 
     region, bucket, key = parse_s3_sqs_payload(payload)
 
-    # For now, we only care about nc files
-    if not key.endswith('.nc'):
-        print(f'No ingest available for key: {key}')
-        return
+    context = PipelineContext(region, DESTINATION_BUCKET_NAME)
 
-    if key_contains(key, NOS_ROMS_FILTERS):
-        output_key = generate_nos_output_key(key)
-        generate_kerchunked_hdf(bucket, key, output_key, DESTINATION_BUCKET_NAME, NOS_DESTINATION_PREFIX)
-    elif key_contains(key, NOS_FVCOM_FILTERS):
-        output_key = generate_nos_output_key(key)
-        generate_kerchunked_netcdf(bucket, key, output_key, DESTINATION_BUCKET_NAME, NOS_DESTINATION_PREFIX)
-    elif key_contains(key, RTOFS_FILTERS):
-        generate_kerchunked_rtofs_nc(region, bucket, key, DESTINATION_BUCKET_NAME, RTOFS_DESTINATION_PREFIX)
-    else:
-        print(f'No ingest available for key: {key}')
\ No newline at end of file
+    # TODO: These could get auto-registered
+    context.add_pipeline('nos_roms', NOS_Pipeline())
+    context.add_pipeline('fvcom', FVCOM_Pipeline())
+    context.add_pipeline('rtofs', RTOFS_Pipeline())
+
+    matching = context.get_matching_pipelines(key)
+    for pipeline in matching:
+        pipeline.run(context.get_region(), bucket, key, context.get_dest_bucket())    
```

Comment:
There's a decision to make here about whether files can trigger only one job. Currently only one pipeline is triggered and then it exits.